packages/services-store/src/actions/__tests__/profileAPI.test.ts

```diff
@@ -50,6 +56,91 @@ describe('Redux actions for `ProfileAPI`', () => {
     ]);
   });
 
+  it('dispatches the action of error in monitoring user authentication status', async () => {
+    ProfileAPI.getUserStatus.mockRejectedValue(new Error('error-getuserstatus'));
+    const store = mockStore();
+    let caught;
+    try {
+      await store.dispatch(loadUserStatus(MASTHEAD_AUTH_METHOD.DEFAULT));
+    } catch (error) {
+      caught = error;
+    }
+    expect(caught?.message).toBe('error-getuserstatus');
+    expect(convertValue(store.getActions())).toEqual([
+      {
+        type: PROFILE_API_ACTION.SET_REQUEST_USER_STATUS_IN_PROGRESS,
+        request: 'PROMISE',
+      },
+      {
+        type: PROFILE_API_ACTION.SET_ERROR_REQUEST_USER_STATUS,
+        error: 'error-getuserstatus',
+      },
+    ]);
+
+    try {
+      await store.dispatch(loadUserStatus(MASTHEAD_AUTH_METHOD.COOKIE));
+    } catch (error) {
+      caught = error;
+    }
+    expect(caught?.message).toBe('error-getuserstatus');
+    expect(convertValue(store.getActions())).toEqual([
+      {
+        type: PROFILE_API_ACTION.SET_REQUEST_USER_STATUS_IN_PROGRESS,
+        request: 'PROMISE',
+      },
+      {
+        type: PROFILE_API_ACTION.SET_ERROR_REQUEST_USER_STATUS,
+        error: 'error-getuserstatus',
+      },
+    ]);
+
+    try {
+      await store.dispatch(loadUserStatus(MASTHEAD_AUTH_METHOD.DOCS_API));
+    } catch (error) {
+      caught = error;
+    }
+    expect(caught?.message).toBe('error-getuserstatus');
+    expect(convertValue(store.getActions())).toEqual([
+      {
+        type: PROFILE_API_ACTION.SET_REQUEST_USER_STATUS_IN_PROGRESS,
+        request: 'PROMISE',
+      },
+      {
+        type: PROFILE_API_ACTION.SET_ERROR_REQUEST_USER_STATUS,
+        error: 'error-getuserstatus',
+      },
+    ]);
+  });
+});
+
+describe('Redux actions for cloud implementation of `ProfileAPI`', () => {
+  it('dispatches the action to set user authentication status', () => {
+    const store = mockStore();
+    store.dispatch(setUserStatus({ user: 'test.user@ibm.com' }));
+    expect(store.getActions()).toEqual([
+      {
+        type: PROFILE_API_ACTION.SET_USER_STATUS,
+        request: { user: 'test.user@ibm.com' },
+      },
+    ]);
+  });
+
```

Comment:
don't think this test is needed it's not specific to the cloud implementation, it's just testing that the redux action is actually setting the correct value in the redux store - there's a test for it up above already
```suggestion
  it('dispatches the action to set user authentication status', () => {
    const store = mockStore();
    store.dispatch(setUserStatus({ user: 'test.user@ibm.com' }));
    expect(store.getActions()).toEqual([
      {
        type: PROFILE_API_ACTION.SET_USER_STATUS,
        request: { user: 'test.user@ibm.com' },
      },
    ]);
  });
```